tests/test_portchannel.py

```diff
@@ -89,6 +89,51 @@ def test_Portchannel(self, dvs, testlog):
         lagms = lagmtbl.getKeys()
         assert len(lagms) == 0
 
+    def test_Portchannel_lacpkey(self, dvs, testlog):
+        portchannelNames = [("PortChannel001", "Ethernet0", 1001),
+                            ("PortChannel002", "Ethernet4", 1002),
+                            ("PortChannel2", "Ethernet8", 12),
+                            ("PortChannel000", "Ethernet12", 1000)]
+
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+
+        # Create 4 PortChannels
+        tbl = swsscommon.Table(self.cdb, "PORTCHANNEL")
+        fvs = swsscommon.FieldValuePairs(
+            [("admin_status", "up"), ("mtu", "9100"), ("oper_status", "up"), ("lacp_key", "auto")])
```

Comment:
can you test without lacp_key, lacp_key = "", lacp_key=n, all four cases?

Reply:
I extended the tests to cover all the lacp_key cases, thank you.